examples/minroot.rs

```diff
@@ -19,7 +19,58 @@ use std::time::Instant;
 use tracing_subscriber::{fmt, prelude::*, EnvFilter, Registry};
 use tracing_texray::TeXRayLayer;
 
-#[derive(Clone, Debug)]
+#[cfg(feature = "abomonate")]
+mod utils {
+  use super::*;
+  use std::{io::Write, mem::size_of};
+
+  pub const FILEPATH: &str = "/tmp/data";
+  /// Unspeakable horrors from a type safety PoV
+  unsafe fn any_as_u8_slice<T: Sized>(p: &T) -> &[u8] {
+    ::core::slice::from_raw_parts((p as *const T) as *const u8, ::core::mem::size_of::<T>())
+  }
+
+  /// this is dangerous
+  #[allow(non_snake_case)]
+  unsafe fn entomb_F<F: PrimeField, W: Write>(f: &F, bytes: &mut W) -> std::io::Result<()> {
+    println!("entomb: {}", size_of::<F>());
```

Comment:
Is this `println` intentional?

Reply:
Yes: most of the example functions through stdout printing, and in so far as we incorporate the demonstration of the abomonate capability, that's how we do it as well.